modal/runner/shared/download.py

```diff
@@ -11,13 +13,16 @@
     models_volume,
 )
 
+logger = get_logger(__name__)
+
 cache_path = get_model_path("__cache__")
 
 downloader_image = (
-    Image.debian_slim()
+    BASE_IMAGE
     # Use the barebones hf-transfer package for maximum download speeds. No progress bar, but expect 700MB/s.
     .pip_install("huggingface_hub==0.19.4")
     .pip_install("hf-transfer==0.1.4")
+    .pip_install("ddtrace==2.4.0")
     .env({"HF_HUB_ENABLE_HF_TRANSFER": "1"})
```

Comment:
This would install on top of the `BASE_IMAGE` instance which is not what we want I think. We should make BASE_IMAGE into a function that return a new Image() instance instead, so that we spawn a fresh image here instead of installing on top of an existing one.

Reply:
adding `ddtrace` here was accidental, removed it.

it looks like they're using the builder pattern to produce these images, in which case building on top of it should be correct here. maybe i'm missing something?

Reply:
Cool beans 👍 - I thought Modal might be doing something weird with how they spawn the image and might need mutation underneath. Confirmed with them that this will make separate instances.